pkg/backup/backup_compaction_test.go

```diff
@@ -422,6 +467,73 @@ func TestBackupCompactionLocalityAware(t *testing.T) {
 	validateCompactedBackupForTables(t, db, []string{"foo"}, collectionURIs, start, end)
 }
 
+func TestScheduledBackupCompaction(t *testing.T) {
+	defer leaktest.AfterTest(t)()
```

Comment:
nice test! how long does it take to run?

Reply:
Takes about 1 second, which is quite amazing given my original implementation was waiting on the order of minutes.